ruby-test-mode.el

```diff
@@ -178,20 +222,20 @@ element of each list in mapping, and replacing the match with the
 second element."
   (let ((target-filename nil))
     (while (and (not target-filename) mapping)
-      (let ((regexp-match (car (car mapping)))
+      (let ((regexp-match (eval (car (car mapping))))
```

Comment:
Oh, I see. Eval is not necessarry when using backquotes as I mentioned above. Eval should be avoided if possible. :)